devito/ir/iet/utils.py

```diff
@@ -66,29 +66,6 @@ def filter_iterations(tree, key=lambda i: i):
     return filtered
 
 
-def compose_nodes(nodes, retrieve=False):
```

Comment:
excellent! it's great when we manage to drop code -- especially if it's old code!